node/node.go

```diff
@@ -824,7 +829,7 @@ func (n *Node) ValidateTransaction(tx *transaction.Transaction) error {
 		core.MaxTxNonceDeltaAllowed,
 	)
 	if err != nil {
-		return nil
+		return fmt.Errorf("%w in Node.ValidateTransaction", err)
```

Comment:
this will arrive in the API response. I don't think a regular user would be interested to know the method where the validation failed. Please add a log for the wrapped error and return it normally

Reply:
done. The error is now logged as warn as it is a programming error (non critical).